virtual-organizations/LIGO.yaml

```diff
@@ -93,3 +93,18 @@ DataFederations:
     AllowedCaches:
       - HCC-Stash
       - Georgia_Tech_PACE_GridFTP2
+      - Stashcache-UCSD
+      - Stashcache-UofA
+      - Stashcache-Cardiff
+      - Stashcache-KISTI
+      - Stashcache-Houston
+      - Stashcache-Manhattan
+      - Stashcache-Sunnyvale
+      - Stashcache-Chicago
+      - Stashcache-Kansas
+      - T1_STASHCACHE_CACHE
+      - PIC_STASHCACHE_CACHE
+
+
+
+
```

Comment:
Since you need to make another change in this file, could you get rid of this extra whitespace?